src/vs/workbench/parts/preferences/browser/keybindingsEditor.ts

```diff
@@ -259,19 +259,49 @@ export class KeybindingsEditor extends BaseEditor implements IKeybindingsEditor
 		this.searchWidget.clear();
 	}
 
-	showSimilarKeybindings(keybindingEntry: IKeybindingItemEntry): TPromise<any> {
-		const value = `"${keybindingEntry.keybindingItem.keybinding.getAriaLabel()}"`;
+	showSimilarKeybindings(keybindingEntry: IKeybindingItemEntry | [ResolvedKeybinding, ResolvedKeybinding]): TPromise<any> {
```

Comment:
I do not think changes to this function are needed, if the events return the `key` string